Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -142,4 +142,26 @@ public struct BuildDescription {
             getBuildTarget(for: $0, in: modulesGraph)
         }
     }
+
+    /// Returns `true` if the file at the given path might influence build settings for a `swiftc` or `clang` invocation
+    /// generated by SwiftPM.
+    public func fileAffectsSwiftOrClangBuildSettings(_ url: URL) -> Bool {
```

Comment:
```suggestion
    @_spi(SwiftPMInternal)
    public func fileAffectsSwiftOrClangBuildSettings(_ url: URL) -> Bool {
```

Reply:
This is part of the SourceKitLSP API, so it should be properly public, not SPI, right? Or should we mark all public methods in the SourceKitLSPAPI module as `@_spi(SwiftPMInternal)` and then do `@_spi(SwiftPMInternal) import SourceKitLSPAPI` in SourceKitLSP?

Reply:
Ugh, sorry, I missed it's a `SourceKitLSPAPI` module, fine for symbols in that to be plain `public`.